### PR TITLE
Feature - Client side Re-evaluation

### PR DESCRIPTION
**Pull request by fzia**

### CODE REVIEW

File: src/Utils/IdentitiesGenerator.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Flagsmith\Utils;
+
+class IdentitiesGenerator
+{
+    public static function generateIdentitiesData(string $identifier, ?object $traits)
```
[the remark this answers is not in the document]
Comment (fzia) [reply src/Utils/IdentitiesGenerator.php:7]: This is being used in src/Flagsmith.php#367.

File: src/Flagsmith.php [hunk 2]
```diff
@@ -22,32 +29,101 @@
 class Flagsmith
 {
     use HasWith;
-
+    public const DEFAULT_API_URL = 'https://api.flagsmith.com/api/v1';
```
[the remark this answers is not in the document]
Comment (fzia) [reply src/Flagsmith.php]: This URL does not seem to be working, However, the change has been made.

File: src/Utils/PollingManager.php [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Flagsmith\Utils;
+
+class PollingManager
```
[the remark this answers is not in the document]
Comment (fzia) [reply src/Utils/PollingManager.php]: Removed this class.

File: src/Utils/IdentitiesGenerator.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (matthewelwell) [reply src/Utils/IdentitiesGenerator.php:7]: Sorry, yes it is, ignore me 🤐

File: examples/templates/index.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    Hello <span id="identifier-name">{{ identifier ? identifier : 'World!'}}</span>,
+  </p>
+  {% if enabled %}
+ <button id="secret-button" style="color: {{ font_colour.colour }};">Secret Button</button>
+{% endif %}
+  <p>
+  <form id="form" method="get">
+    <h3>Identify as a user</h3>
+    <label for="identifier">Identifier: </label>
+    <input name="identifier" id="identifier" />{{ identifier ? identifier : ''}}
```
Comment (matthewelwell) [inline examples/templates/index.html:22]: I don't think we need the additional `{{ identifier ? identifier : '' }}` piece here?

File: examples/examples.php [hunk 5]
```diff
@@ -0,0 +1,45 @@
+<?php
```
Comment (matthewelwell) [inline examples/examples.php:1]: Do we need this file at all? This is the one that just prints a load of stuff out, right?

**Review by matthewelwell — APPROVED** [review]

**Pull request merged**